website/content/docs/concepts/connection-workflows/exec-flag.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: -exec flag
-description: |-
-  Learn how the `-exec` flag enables you to execute Boundary TCP sessions using your preferred client, even when there is no built-in support for it.
+description: >-
+  Learn how to use the `-exec` flag to execute TCP sessions or pass flags using your preferred client, even when there is no built-in support for it.
```

Comment:
curious if you think this part is useful?

> even when there is no built-in support for it.

I'm curious how in-depth we should make the description, since it's never exposed to the reader. Will this statement improve SEO or discoverability?

Reply:
No, it probably isn't. I removed it. Thanks!